loggers/loggers.go

```diff
@@ -0,0 +1,53 @@
+package loggers
+
+import (
+	"io"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// MT a logger that is created via the LoggerManager and is thread safe (MT == multi-threaded)
+type MT struct {
+	*log.Logger
+}
+
+// LoggerManager a manager that can produce loggers that are synchronized internally
+type LoggerManager struct {
+	internalWriter ThreadSafeWriter
+}
+
+// AdaptLogger will take a logger instance and wrap it for synchronization
+func AdaptLogger(log *log.Logger) *LoggerManager {
+	return MakeLoggerManager(log.Out)
+}
```

Comment:
I don't think you need this function:
* If `MakeLogger` returns log.Logger, then you don't need to change all the interfaces.
* Now you don't need `MakeLoggerManager` to wrap the writer, all the tests could continue using `log.New`.